opentech/apply/determinations/tests/test_views.py

```diff
@@ -126,6 +126,44 @@ def test_can_progress_stage_via_determination(self):
         self.assertEqual(submission_original.status, 'invited_to_proposal')
         self.assertEqual(submission_next.status, 'draft_proposal')
 
+    def test_accepted_determination_creates_project(self):
+        submission = ApplicationSubmissionFactory(
+            status='concept_review_discussion',
+            workflow_stages=2,
+            lead=self.user,
+        )
+
+        self.post_page(submission, {
+            'data': 'value',
+            'outcome': ACCEPTED,
+            'message': 'You are invited to submit a proposal',
+        }, 'form')
+
+        # Cant use refresh from DB with FSM
+        submission_original = self.refresh(submission)
+        submission_next = submission_original.next
+
+        self.assertTrue(hasattr(submission_next, 'project'))
+
+    def test_non_accepted_determination_does_not_create_project(self):
+        submission = ApplicationSubmissionFactory(
+            status='concept_review_discussion',
```

Comment:
this should be `proposal_review_discussion` the `concept_review_discussion` would fall foul of the `is_final_stage` check, which i cant see being tested anywhere?

Reply:
What's the difference between these two statuses?

Reply:
stage 1: `concept_review_discussion` -> `is_final_stage` = False
stage 2: `proposal_review_discussion`-> `is_final_stage` = True

Reply:
Fixed and rebased.